getResource_test.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestFoldersTableSlice(t *testing.T) {
+	var foldersResBody *foldersResp
```

Comment:
It's a little unclear what you're actually testing here. It looks like you're mostly testing the built-in `json.Unmarshal` functionality, which is fine as a sanity check (just to double-check that your types are sane), but isn't really that valuable overall. You should really try to mostly test your actual application logic; I think a "higher-level" test would probably make more sense.